src/Sulu/Bundle/SecurityBundle/Command/CreateUserCommand.php

```diff
@@ -136,6 +139,20 @@ protected function getUser()
     }
 
     /**
+     * Returns true if the passed email is a unique user-email
+     * @param $email
+     * @return bool
+     */
+    protected function isEmailUnique($email) {
```

Comment:
@marcelmoosbrugger is there an issue for that? or why do you include this in this PR?